pandas/tseries/frequencies.py

```diff
@@ -422,6 +422,27 @@ def get_period_alias(offset_str):
     return _offset_to_period_map.get(offset_str, None)
 
 
+_pure_alias = {
```

Comment:
why is this necessary? why is this not added to the ``_offsset_to_period_map`` and creating new code?